_posts/payalpatel/2016-02-03-Ch5Exercises.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+author: payalpn
+title: "Payal's Chapter 5 Loops Exercises"
+---
+
+**Chapter 5, Exercise 1:**
+
+<iframe src="https://trinket.io/embed/python/0c8682b715" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+**Chapter 5, Exercise 2:**
+
+<iframe src="https://trinket.io/embed/python/5a5b5b49bc" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+**Reflection:**
+
+At first I thought both of these exercises were difficult.  I was having trouble starting, so in order to deal with this I split up the problems into divisible and more manageable tasks.  I then arranged them in an order that I thought made logical sense (in terms of completing each of these smaller subtasks).  For me this helped a lot—I was able to focus on smaller tasks and then piece them together once I had finished the components.  
```

Comment:
Awesome- this strategy will pay huge dividends as our programs get more and more complex.